operator/src/main/java/org/keycloak/operator/controllers/KeycloakUpdateJobDependentResource.java

```diff
@@ -194,24 +200,11 @@ private static void configureContainer(ContainerFluent<?> containerBuilder, Stri
                 .endVolumeMount();
     }
 
-    private Map<String, Volume> getAllVolumes(Context<Keycloak> context) {
-        Map<String, Volume> allVolumes = new HashMap<>();
-        Consumer<Volume> volumeConsumer = volume -> allVolumes.put(volume.getName(), volume);
-        CRDUtils.volumesFromStatefulSet(ContextUtils.getCurrentStatefulSet(context).orElseThrow()).forEach(volumeConsumer);
-        CRDUtils.volumesFromStatefulSet(ContextUtils.getDesiredStatefulSet(context)).forEach(volumeConsumer);
-        return allVolumes;
-    }
-
-
     private static List<String> replaceStartWithUpdateCommand(List<String> currentArgs, List<String> updateArgs) {
-        return currentArgs.stream().
-                <String>mapMulti((arg, downstream) -> {
-            if (START_ARGS.contains(arg)) {
-                updateArgs.forEach(downstream);
-                return;
-            }
-            downstream.accept(arg);
-        }).toList();
+        // TODO: reuse ConfigArgConfigSource parsing
+        // also start-dev is problematic - at the very least --profile=dev should be added
```

Comment:
I wonder how a `start-dev` would end up here ... again via a `podTemplate`? I'd say this is not supported when using the Operator?

Reply:
Yes, the podTemplate is the only way to use start-dev.

If we are ok with letting the job fail to start, then I can just minimize this comment and move on - this in general is a misconfiguration because you'd have to go to great lengths to cluster in dev mode.

Reply:
I'm ok with letting it fail to start.